schemas/7/vault.json

```diff
@@ -16,178 +17,53 @@
       "format": "uuid"
     },
     "email": {
-      "type": ["string", "null"],
-      "default": null,
+      "type": "string",
       "format": "idn-email"
     },
     "validatedPhoneNumber": {
-      "type": ["string", "null"],
-      "default": null
+      "type": "string"
     },
     "manualItems": {
       "type": "array",
       "items": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "string",
-            "format": "uuid"
-          },
-          "name": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "url": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "username": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "password": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "intendedPassword": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "otpSeed": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "notes": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "relatedItems": {
-            "type": "array",
-            "items": {}
-          },
-          "ssoProvider": {
-            "type": ["array", "null"],
-            "items": {}
-          }
-        },
-        "required": ["id"]
+        "$ref": "#/$defs/manualItem"
       }
     },
     "vault": {
       "type": "array",
       "items": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "string",
-            "format": "uuid"
-          },
-          "name": {
-            "type": "string"
-          },
-          "username": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "password": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "intendedPassword": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "intendedOTPSeed": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "otpSeed": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "notes": {
-            "type": ["string", "null"],
-            "default": null
-          },
-          "relatedItems": {
-            "type": "array",
-            "items": {}
-          },
-          "ssoProvider": {
-            "type": ["array", "null"],
-            "items": {}
-          }
-        },
-        "required": ["id", "name"]
+        "$ref": "#/$defs/vault"
```

Comment:
what do you think about calling these `vaultItem` instead of `vault`?

Reply:
yes good idea